modules/apps/foundation/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/dynamic_inline_scroll.js

```diff
@@ -0,0 +1,161 @@
+AUI.add(
+	'liferay-dynamic-inline-scroll',
+	function(A) {
+		var Lang = A.Lang;
+
+		var TPL_LIST_ITEM = '<li>' +
+				'<a href="{href}" onclick="{onlick}">{pageTitle}</a>' +
+			'</li>';
+
+		var DynamicInlineScroll = A.Component.create(
+			{
+				ATTRS: {
+					cur: {
+						value: 0
+					},
+
+					curParam: {
+						value: ''
+					},
+
+					forcePost: {
+						value: false
+					},
+
+					formName: {
+						value: ''
+					},
+
+					initialPages: {
+						value: 0
+					},
+
+					jsCall: {
+						value: ''
+					},
+
+					namespace: {
+						value: ''
+					},
+
+					pages: {
+						value: 0
+					},
+
+					url: {
+						value: ''
+					},
+
+					urlAnchor: {
+						value: ''
+					}
+				},
+
+				NAME: 'liferaydynamicinlinescroll',
+
+				prototype: {
+					initializer: function() {
+						var instance = this;
+
+						instance._inlineScrollers = A.all('ul.pagination ul.inline-scroller');
```

Comment:
Did you have a particular use case or test case in mind?

I tested this by adding another (identical) paginator to the page via the JSPF (html/taglib/ui/search_iterator/lexicon/bottom.jspf), and all the inline scrollers worked as expected.